Formula/o/opentdf-cli.rb

```diff
@@ -0,0 +1,52 @@
+class OpentdfCli < Formula
+  desc "OpenTDF command-line interface"
+  homepage "https://docs.opentdf.io/"
+  url "https://github.com/opentdf/otdfctl.git",
+      tag:      "v0.11.4",
+      revision: "1e64065d08547658e436dc167b108c2a6e9c0ccd"
+  license "BSD-3-Clause-Clear"
+  head "https://github.com/opentdf/otdfctl.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_releases
+  end
+
+  keg_only :versioned_formula
+
+  # disable! date: "2025-01-01", because: :deprecated_upstream
+
+  depends_on "bash" => :build
+  depends_on "go" => :build
+
+  uses_from_macos "rsync" => :build
+
+  on_macos do
+    depends_on "coreutils" => :build
+  end
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/opentdf/otdfctl/pkg/config.Version=#{version}
+      -X github.com/opentdf/otdfctl/pkg/config.BuildTime=#{time.iso8601}
+      -X github.com/opentdf/otdfctl/pkg/config.CommitSha=#{Utils.git_head}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags:), "-o", bin/"otdfctl"
```

Comment:
Why not the formula name?

Reply:
Would the preferred approach be to keep the formula name and add an alias as I did? 

My goal is to make sure discovery of OpenTDF tools are high while supporting the conventional name of `otdfctl`.

Reply:
Looking deeper, I see that 144 formulas that follow this pattern (although I cannot quickly derive the number that don't name the binary after the `*-cli` pattern.

https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-core+path%3AFormula%2F*%2F*-cli.rb&type=code

Reply:
I think you can just do `otdfctl`, it would error out when people installing `otdfctl-cli` and recommend `otdfctl`.

the fact there are other aliases might be due to the repo renaming or artifact renaming reasons historically, not from the beginning.

Reply:
That makes sense and it seems if I have a discovery issue in the future I can resurface that as a discussion.